ext/opentelemetry-ext-jaeger/src/opentelemetry/ext/jaeger/__init__.py

```diff
@@ -153,8 +156,19 @@ def _translate_to_jaeger(spans: Span):
 
         tags = _extract_tags(span.attributes)
 
-        # TODO: status is missing:
-        # https://github.com/open-telemetry/opentelemetry-python/issues/98
+        if tags is None:
+            tags = []
```

Comment:
It might be easier to change `_extract_tags` to always return an empty list, rather than checking this here

Reply:
And in fact we probably should since the only other place where it's used also ends up doing some juggling around this: https://github.com/open-telemetry/opentelemetry-python/blob/ed8cbdc74b24dd2d9b30477b054509c1f81a94f6/ext/opentelemetry-ext-jaeger/src/opentelemetry/ext/jaeger/__init__.py#L239-L243

Reply:
Fixed! Added the check when `span.status` wasn't mandatory, but now It makes no sense.

Reply:
mind updating the in `_extract_logs_from_span` as well?

Reply:
But if we change `_extract_logs_from_span`  to return an empty list when a span has no logs tests break because they expect logs to be None. I think that it's the same with [refs](https://github.com/open-telemetry/opentelemetry-python/blob/ed8cbdc74b24dd2d9b30477b054509c1f81a94f6/ext/opentelemetry-ext-jaeger/src/opentelemetry/ext/jaeger/__init__.py#L198).

```
# Example output form tests
Span([348 chars]Double=None, vBool=None, vLong=None, vBinary=None)], logs=[])
Span([348 chars]Double=None, vBool=None, vLong=None, vBinary=None)], logs=None)
```

Maybe I'm misunderstanding something. What do you think?

Reply:
Sorry, I meant to update the code in `_extract_logs_from_span` to take advantage of this new empty list from `_extract_tags`, lines 239 and 240 are no longer needed:
https://github.com/open-telemetry/opentelemetry-python/blob/ed8cbdc74b24dd2d9b30477b054509c1f81a94f6/ext/opentelemetry-ext-jaeger/src/opentelemetry/ext/jaeger/__init__.py#L239-L243

Reply:
Oh, Okay! It should be updated now.